.tests/integration/configs/fastp_merge_config.yaml

```diff
@@ -1,9 +1,5 @@
 samples: "samples.tsv"
 units: "units.tsv"
 
-default_container: "docker://marrip/common:1.1.0"
```

Comment:
I am a bit confused. You removed all container information from your `config.yaml`. Are you using conda instead?